README.md

```diff
@@ -1 +1,106 @@
-# armo-platform-tools
\ No newline at end of file
+# ARMO POC Prerequisite Validation Script
+
+This script is designed to validate the prerequisites for the ARMO Security Proof of Concept (POC). It performs the following checks:
+
+1. Network accessibility
```

Comment:
I don't see why you need to check that?

Reply:
This is a requirement from a prospect in order to be ready to the POC kickoff.